honeybee/model.py

```diff
@@ -1349,6 +1375,21 @@ def to_dict(self, included_prop=None, triangulate_sub_faces=False):
 
         return base
 
+    @staticmethod
+    def to_json(hb_dict, folder_path, name="unnamed"):
```

Comment:
The arguments here should match those of the `to_dict` method on the model object but with a couple of new ones specifically for the JSON serialization.  I think a single `file_path` argument is probably better than separate arguments for folder and name. It's probably also good to include an `indent` argument as this is pretty useful.